frontend/app/(footer)/page.tsx

```diff
@@ -1,51 +1,299 @@
+// "use client";
```

Comment:
we don't want duplicate code, best way is to simply add a redirect using next.js config. We don't have to comment out or duplicate code